src/test/java/com/google/sps/lib/algorithm/distance/MapsApiDistanceCalculatorTest.java

```diff
@@ -26,6 +26,9 @@
 import java.net.HttpURLConnection;
 import org.mockito.Mock;
 import org.mockito.runners.MockitoJUnitRunner;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Arrays;
 
 @RunWith(MockitoJUnitRunner.class)
 public class MapsApiDistanceCalculatorTest {
```

Comment:
Please add one more test case which checks that the request URL is correct for a given set of coordinates.